### ISSUE TITLE
Porn Link

### ISSUE THREAD

**Issue by jakiestfu** [zalmoxisus/remote-redux-devtools#152]
Hello. The link to http://remotedev.io/local currently redirects to pornogrophy.

NSFW. This is linked to directly within the readme file.

**Comment by FDiskas** [comment]
And it still does.

**Comment by Methuselah96** [comment]
All the remaining references to `remotedev.io` should have been removed in https://github.com/zalmoxisus/remote-redux-devtools/pull/154. Did I miss any?

**Comment by FDiskas** [comment]
Yes there is more references in other repose like https://github.com/zalmoxisus/remote-redux-devtools

**Comment by Methuselah96** [comment]
I'm not sure what you mean. The repository you linked to is this repository.

**Comment by Methuselah96** [comment]
Got it, I don't have write access to that repository.
